.gitignore

```diff
@@ -6,6 +6,7 @@
 
 /prometheus
 /promtool
+/tsdb/tsdb
```

Comment:
The tsdb binary is actually built at `/tsdb/cmd/tsdb/tsdb`

Reply:
I tried `make build` and it generates /tsdb/tsdb by promu:
```
➜  prometheus git:(master) ✗ make build
>> writing assets
# Un-setting GOOS and GOARCH here because the generated Go code is always the same,
# but the cached object code is incompatible between architectures and OSes (which
# breaks cross-building for different combinations on CI in the same container).
cd web/ui && GO111MODULE=on GOOS= GOARCH= go generate -x -v  -mod=vendor
doc.go
go run -mod=vendor assets_generate.go
writing assets_vfsdata.go
ui.go
>> building binaries
GO111MODULE=on /Users/kevinkfeng/code/go/bin/promu build --prefix /Users/kevinkfeng/code/go/src/github.com/prometheus/prometheus
 >   prometheus
 >   promtool
 >   tsdb
```
Is there something wrong?

Reply:
Apparently yes. `make build_tsdb` seems to build it in `/tsdb/cmd/tsdb/tsdb`. Could you modify `build_tsdb` to also build as `/tsdb/tsdb` to be consistent (@krasi-georgiev sounds good?) and incorporate https://github.com/prometheus/prometheus/pull/6204 in it?